docs/other_resources/in-process-providers/specification.md

```diff
@@ -0,0 +1,247 @@
+# Creating an in-process flagd provider
+
+By default, **flagd** is a remote service that is accessed via **grpc** by a client application to retrieve feature flags.
+Depending on the environment, flagd therefore is usually deployed as a standalone service, e.g. as a Kubernetes Deployment,
+or injected as a sidecar container into the pod running the client application,
+as it is done in the [OpenFeature Operator](https://github.com/open-feature/open-feature-operator).
+An in-process flagd provider, on the other hand, is designed to be embedded into the application and therefore
+no communication outside the process of the application is needed. This can be desired by some architectures,
```

Comment:
```suggestion
no communication outside the process of the application for feature flag evaluation is needed. This can be desired by some architectures,
```